addons/bus/tests/test_notify.py

```diff
@@ -55,6 +57,8 @@ def check_payloads_size(payloads):
 
     def test_postcommit(self):
         """Asserts all ``postcommit`` channels are fetched with a single listen."""
+        if ODOO_NOTIFY_FUNCTION != 'pg_notify':
+            return
```

Comment:
Can't we import and patch `ODOO_NOTIFY_FUNCTION` from bus to ensure it is always `pg_notify` in test instead?

Reply:
> Can't we import and patch `ODOO_NOTIFY_FUNCTION` from bus to ensure it is always `pg_notify` in test instead?

That's the point of skipping the test. The LISTEN won't work when this option is being used.

Reply:
@seb-odoo they are adding it for their own runbot at odoosh, I don't think it is going to change anything for us, am I right @sts-odoo?

Reply:
@seb-odoo @Julien00859 indeed